packages/sass-variables/_typography.scss

```diff
@@ -32,56 +32,91 @@ $puppet-type-colors: (
   success: $puppet-type-color-success,
 );
 
+// Hero Heading
 $puppet-type-hero-font-size: 48px !default;
-$puppet-type-hero-line-height: calc(56 / 48) !default;
+$puppet-type-hero-line-height: calc(60 / 48) !default;
 $puppet-type-hero-font-family: $puppet-type-font-family-secondary !default;
 $puppet-type-hero-font-weight: $puppet-type-weight-heavy !default;
+$puppet-type-hero-margin: 32px !default;
```

Comment:
Wasn't totally sure whether to add these margins as variables, but I could imagine using them if I was making a custom component in a separate project, so I went with it.